meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -134,18 +134,20 @@ it5570_shutdown() {
 }
 
 check_network_connection() {
-  echo 'Waiting for network connection ...'
-  n="10"
+  echo 'Waiting for network connection...'
+  n="5"
```

Comment:
Why do we also change the number of iterations here?

Reply:
Noted. Reverted the change and squashed commits